script.js

```diff
@@ -4,3 +4,103 @@
 // - A User should only be able to select one box per turn
 // - A User should not be able to select a box that is already selected
 // - When game is over (whether a win or tie) the users should be prompted to play again and the game should be reset
+
+let gamePiece = "";
+let playerTurn = true;
+let text = document.getElementById("text");
+const xGamePiece = "X";
+const oGamePiece = "O";
+const gameSpaces = document.querySelectorAll(".gameSpace");
+
+gameSpaces.forEach(addEventListenerToGameSpaces);
+
+function addEventListenerToGameSpaces(gameSpace) {
+  gameSpace.addEventListener("click", useGamePiece);
+}
+
+text.innerHTML += "Player X turn";
```

Comment:
will there be older text we want to keep and add this on to? Or is this just replacing the empty string?